CONTRIBUTING.md

```diff
@@ -324,28 +122,156 @@ In order for a PR to be merged, the following criteria should be met:
+### Documenting your change<a name="documenting-your-change"></a>
 
-[1]
-```
-sample markdown for checklist
+When there is a doc impact on your change, you have two choices:
+
+1. Submit an issue to [Docs issues](https://github.com/rackerlabs/docs-rpc/issues)
+OR
+2. Contribute your own doc change to [docs-rpc](https://github.com/rackerlabs/docs-rpc) and tag a relevant docs person for review and merge.
+
+### What to document<a name="what-to-document"></a>
+
+Document changes if:
 
----
-- [x] master PR: https://github.com/rcbops/rpc-openstack/pull/{{number}}
-- [] kilo PR: https://github.com/rcbops/rpc-openstack/pull/{{number}}
+* There are changes to the way playbooks are run/named
+* Changes to the documented configuration files
+* New feature
+* Change that requires you (as the developer) to take action
+* Update impact changes
+
+### How to document<a name="how-to-document"></a>
+
+The documentation work is all done in [this](https://github.com/rackerlabs/docs-rpc) repo.
+
+1. See the [Docs workflow](https://github.com/rackerlabs/docs-rpc/blob/master/GITHUBING.rst) to setup for contribution and
+   [here](https://github.com/rackerlabs/docs-rpc/blob/master/CONTRIBUTING.rst) for our Contributor guidelines.
+2. Once you have created a pull request, tag the doc team for review. See [Documentation team FAQ](https://github.com/rackerlabs/docs-rackspace/blob/master/doc/contributor-collateral/privatecloud-docteam-FAQ.rst) for clarification on who to contact.
+3. Await merge. Respond to any questions/comments/concerns.
+
+Any further questions about our github processes, contributor guidelines, or FAQ about the docs and team, refer to: [Contributor collatoral](https://github.com/rackerlabs/docs-rackspace/tree/master/doc/contributor-collateral)
+
+## Enabling an upstream project in RPCO<a name="enabling-upstream-project"></a>
+
+### Integrating with OpenStack-Ansible<a name="integrating-with-osa"></a>
+
+Before an OpenStack feature can be integrated into RPCO, it must first be implemented into the OpenStack-Ansible(OSA) project.
+For documentation on how to add a service and Ansible role to OpenStack-Ansible, please see [this page](http://docs.openstack.org/developer/openstack-ansible/developer-docs/additional-roles.html).
+
+Once the new project is fully integrated into OpenStack-Ansible, it might not be consumable by RPCO yet. Since RPCO and OSA do not have matching developing cycles, the project will either have to be backported to a stable branch in OSA(unlikely) or will not be included until RPCO moves on to the release with the new project. For example, if the new project is added into the master branch of OSA(currently Otaca), it will not be consumable by RPCO until RPCO's master branch has been modified to consume OSA's Otaca release. It is recommended to start this integration into OSA 6-12 months before you intend on releasing the feature in RPC.
```

Comment:
Because some of these are really long lines, my single line comment doesn't appear right next to the requested change. LMK if you need clarification on any of them.

(also, please consider line wrapping these docs at 72 chars. pep8 has a fairly good standard https://www.python.org/dev/peps/pep-0008/#maximum-line-length)